podcast/2.14.md

```diff
@@ -0,0 +1,601 @@
+I got into computer programming in what I feel like was a very popular way at a particular point in time. I liked video games as a kid, and I wanted to write video games, so I learned how to program. Then, I quickly discovered that writing video games is hard, and it wasn't actually something I was naturally all that good at, but I really enjoyed the programming part. I voraciously took off from there, learning as much about software engineering and programming and computers as I could, and I haven't really stopped since.
+
+**Beyang Liu:**
+
+That's cool. What was the video game that originally got you interested? Do you remember?
+
+**Nelson Elhage:**
+
+I don't remember if there was a specific one or what. I grew up on the Nintendo 64, and that was my formative platform as a child that I played a lot of games on and learned the joy of gaming. I love Nintendo games to this day. Then also, I think one of the other platforms that really got me was the TI calculators. We got the TI calculators in school at some point, the TI-83 Plus, I think. I had to do math with it, but those are programmable in their own dialect of BASIC, and so I started learning to write games there. I would program on them during class when I was bored or distracted.
+
+I actually eventually learned a bit of assembly programming very early on because the other language that you can program the TI calculators in is by writing Z80 Assembly against TI's OS. That's the only way to get high performance, so all the really good games were written in that. I started learning that without having any idea what I was doing at that point, but that was a lot of fun.
+
+**Beyang Liu:**
+
+That's awesome. I actually got into programming the same way. I had a TI-83 Plus, and I was fortunate enough to get one with the manual. In the manual, they had a BASIC tutorial that I read through, but I never made it to the assembly, so you were operating at another level. I topped out at just the BASIC.
+
+**Nelson Elhage:**
+
+The assembly was a high-stakes game because there was no memory protection on those things, so any mistakes would reboot the calculator. I think it was even relatively possible to completely break it and require a factory reset of some sort–especially without having any idea what you're doing. It was a very slow process. Often, you'd try to write something and it would fail. You wouldn't have any idea why it failed, but you'd have to go take out the batteries and the backup battery, reset the RAM, and start over.
+
+**Beyang Liu:**
+
+Do you remember any of the programs you wrote? Do you have any crowning achievements that you're super proud of on that platform?
+
+**Nelson Elhage:**
+
+Honestly, I feel like the weird way that I was most proud of was one of the very first I did. I built a trivial whack-a-mole game, where little moles would appear up, and you had to whack them using the number pad as a three-by-three grade corresponding to a three-by-three grade on the screen.
```

Comment:
@nickmyyz I'm wondering if this should have been "weird one"? It doesn't read right as is